ansible_collections/f5networks/f5_modules/plugins/modules/bigip_imish_config.py

```diff
@@ -629,18 +630,26 @@ def load_config_on_device(self, name):
             self.client.provider['server'],
             self.client.provider['server_port']
         )
-        resp = self.client.api.post(uri, json=params)
-        try:
-            response = resp.json()
-        except ValueError as ex:
-            raise F5ModuleError(str(ex))
 
-        if resp.status in [200, 201] or 'code' in response and response['code'] in [200, 201]:
-            if 'commandResult' in response:
-                if 'Dynamic routing is not enabled' in response['commandResult']:
-                    raise F5ModuleError(response['commandResult'])
-            return True
-        raise F5ModuleError(resp.content)
+        x = 0
+        while x < 3:
+            resp = self.client.api.post(uri, json=params)
+            try:
+                response = resp.json()
+            except ValueError as ex:
+                raise F5ModuleError(str(ex))
+
+            if resp.status in [200, 201] or 'code' in response and response['code'] in [200, 201]:
+                if 'commandResult' in response:
+                    if 'Dynamic routing is not enabled' in response['commandResult']:
+                        raise F5ModuleError(response['commandResult'])
+                    if 'Protocol daemon is not running' in response['commandResult']:
+                        x += 1
```

Comment:
HI @urohit011 ,
do you think 15 seconds of sleep is enough, is there any scenario it takes more time

Reply:
so far it seems that 5 seconds are enough.